source/classes/sc_exception.h

```diff
@@ -39,6 +39,18 @@ sc_exception::sc_exception(ic_string *msg, int type, const char *file, long line
 }
 
 /**
+ * sc_exception copy constructor.
+ * @param original The original object to copy.
+ */
+sc_exception::sc_exception(const sc_exception &original)
```

Comment:
Why do we need the copy constructor? Is it used anywhere?

Reply:
That's a good question. Actually, copy constructor is necessary for every type we throw by value (it _may_ be unused by the compiler, but its presence is obligatory). Before my changes, it was defined implicitly and used when we were copying `sc_exception` in one case.

Now we aren't explicitly using it anywhere, but it still should be defined.